src/hotspot/share/opto/vectornode.cpp

```diff
@@ -937,7 +1005,7 @@ Node* StoreVectorMaskedNode::Ideal(PhaseGVN* phase, bool can_reshape) {
       }
     }
   }
-  return NULL;
+  return StoreNode::Ideal(phase, can_reshape);
```

Comment:
Should this call `StoreVectorNode::Ideal`?

Reply:
ditto